python_modules/dagster/dagster/_core/definitions/asset_condition/asset_condition.py

```diff
@@ -31,6 +31,7 @@
 
 if TYPE_CHECKING:
     from .asset_condition_evaluation_context import AssetConditionEvaluationContext
+    from .scheduling_condition_evaluation_context import SchedulingConditionEvaluationContext
```

Comment:
We can rename in a sep diff but i think we can shorten this to `SchedulingContext`

Reply:
Added a diff upstack to do this rename